flytekit/core/base_sql_task.py

```diff
@@ -41,7 +41,7 @@ def __init__(
             task_config=task_config,
             **kwargs,
         )
-        self._query_template = query_template
+        self._query_template = query_template.replace("\n", "\\n").replace("\t", "\\t")
```

Comment:
are there any other characters or escape sequences that are specific to snowflake? I tried to read https://docs.snowflake.com/en/sql-reference/data-types-text.html, but I'm still not sure if we have to handle some of those.

Reply:
I feel like some of those can handle by users themself. if we triple quote like
```
 query = """select 1
 from table"""
```
python will convert it to `select 1\nfrom table`, so that's why we help them handle this special character in flytekit.

However, if they want to run something like `select $1, $2 from ('Backslash','C:\user');`, they are able to explicit convert it to `select $1, $2 from ('Backslash','C:\\user');` by themself to make it work in snowflake.